code/datums/uplink_item.dm

```diff
@@ -621,6 +621,12 @@ var/list/uplink_items = list()
 	item = /obj/item/toy/balloon/decoy
 	cost = 1
 
+/datum/uplink_item/stealthy_tools/flashlightemp
+	name = "EMP Flashlight"
+	desc = "A flashlight that blasts a weak EMP pulse on whatever or whoever you use it on. Up to 4 charges that recover every 15 seconds, as shown when examined. Devastating against energy weapons and silicons. Can use it to cheat at the Arcade machine."
```

Comment:
30 seconds, not 15 seconds

Reply:
Timer shows 15 though, even though I know it takes 30 seconds, and I just thought of an idea

Reply:
remember that we have the `SECONDS` macro

Reply:
Don't really know where to put SECONDS because charge_tick is counted in a process that happens only every tick, at least I don't know how to add it without making another var